src/main/java/org/apache/maven/plugins/clean/Cleaner.java

```diff
@@ -273,7 +273,8 @@ private boolean isSymbolicLink(Path path) throws IOException {
      * @throws IOException If a file/directory could not be deleted and <code>failOnError</code> is <code>true</code>.
      */
     private int delete(File file, boolean failOnError, boolean retryOnError) throws IOException {
-        if (!file.delete()) {
+        IOException error = delete(file);
```

Comment:
this is not a Java error; please rename the variable

Reply:
Will do.
To justify why I named it like this: it was to be consistent with the context of the existing terminology `failOnError` - it is the `error` in that context.

Reply:
I renamed it to `failure`.